src/main/java/org/dspace/ctask/replicate/store/DuraCloudObjectStore.java

```diff
@@ -164,7 +163,8 @@ public long transferObject(String group, File file) throws IOException
         }
         catch (ContentStoreException csE)
         {
-            throw new IOException(csE);
+            // Log the DuraCloud exception message for getContentProperties.
```

Comment:
Should we not be throwing this information up to the task code so that it can be added to the report?

Reply:
Agree with both comments.  I was late noticing the task report function and had cooked this up already.  It would be better just handle the report in the task.